canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -1630,6 +1630,22 @@ export default class PropertiesController {
 		return this.propertiesStore.getSaveButtonDisable();
 	}
 
+	/**
+	* Enable/disable OK button for given summary panel
+	* @param panelId {name: panel.id}
+	* @param okDisable boolean
+	*/
+	setOkButtonDisable(panelId, okDisable) {
```

Comment:
Would this be better named `setPrimaryButtonDisabled`?

Reply:
Should I rename it to `setWideFlyoutPrimaryButtonDisabled` ?
`setPrimaryButtonDisabled`  may be confused with an existing method `setSaveButtonDisable`.

Reply:
That's probably better.